src/Migrator.php

```diff
@@ -272,22 +288,20 @@ private function generateTableTriggerSql($table, Connection $sourceConnection)
                 foreach ($triggers as $trigger) {
                     $sql = 'CREATE TRIGGER ' . $trigger['Trigger'] . ' ' . $trigger['Timing'] . ' ' . $trigger['Event'] .
                         ' ON ' . $sourceConnection->quoteIdentifier($trigger['Table']) . ' FOR EACH ROW ' .PHP_EOL . $trigger['Statement'] . '; ';
-                    $createTriggersSql[] = $sql;
+                    $this->tableTriggers[] = $sql;
                 }
             }
         } elseif ($driverName === 'pdo_sqlite') {
             $schemaSql = "select sql from sqlite_master where type = 'trigger' AND tbl_name=" . $sourceConnection->quote($table);
             $triggers = $sourceConnection->fetchAll($schemaSql);
             if ($triggers && count($triggers) > 0) {
                 foreach ($triggers as $trigger) {
-                    $createTriggersSql[] = $trigger['sql'];
+                    $this->tableTriggers[] = $trigger['sql'];
                 }
             }
         } else {
             throw new \RuntimeException(__METHOD__ . " not implemented for $driverName yet");
         }
-
-        return $createTriggersSql;
```

Comment:
I think this should still `return`? (side-effects make me uneasy...)

Reply:
Refactored to return the trigger SQL to maintain congruency with the function name